README.md

```diff
@@ -46,23 +46,26 @@ Receptor is hosted on AWS at *(In progress)*
          ```
          brew services start postgresql
          ```
-
+     4. To get Python working with Postgres, install psycopg2
+          ```
+          pip install psycopg2
+          ```
```

Comment:
I believe you need this step when running on a different OS. Perhaps move this step outside <details> before the step number 5?

Reply:
This was a needed step for me on M1 macOS12.0. How about I move this to a separate step? Something like this
`To get Python working with Postgres, install psycopg2. This will be required on M1 mac OS12.0 & above.`

Reply:
On another note, why not add this to requirements.txt file instead?

Reply:
That's a good idea. I'll add it requirements.txt.